src/main/java/org/cryptomator/ui/preferences/UpdatesPreferencesController.java

```diff
@@ -77,11 +120,66 @@ public String getCurrentVersion() {
 		return currentVersion;
 	}
 
-	public BooleanBinding updateAvailableProperty() {
+	public ObservableValue<Instant> lastSuccessfulUpdateCheckProperty() {
+		return lastSuccessfulUpdateCheck;
+	}
+
+	public String getLastSuccessfulUpdateCheck() {
+		Instant lastCheck = lastSuccessfulUpdateCheck.getValue();
+		if (lastCheck != null && !lastCheck.equals(Settings.DEFAULT_TIMESTAMP)) {
+			return formatter.format(LocalDateTime.ofInstant(lastCheck, ZoneId.systemDefault()));
+		} else {
+			return "-";
+		}
+	}
```

Comment:
Property and getter are of different type. Apparently this works, but it shouldn't. From the FXML side we want to bind to the string representation, so what we should do here is this:

```java
// fields:
ObservableValue<Instant> lastSuccessfulUpdateCheck;
StringProperty lastUpdateCheckMessage;

// in constructor:
lastUpdateCheckMessage = Bindings.createStringBinding(this::getLastUpdateCheckMessage, lastSuccessfulUpdateCheck);
```

```suggestion
	public StringBinding lastUpdateCheckMessageProperty() {
		return lastUpdateCheckMessage;
	}

	public String getLastUpdateCheckMessage() {
		Instant lastCheck = lastSuccessfulUpdateCheck.getValue();
		if (lastCheck != null && !lastCheck.equals(Settings.DEFAULT_TIMESTAMP)) {
			return formatter.format(LocalDateTime.ofInstant(lastCheck, ZoneId.systemDefault()));
		} else {
			return "-";
		}
	}
```

Note that the `lastSuccessfulUpdateCheck` can be private, as I don't see any external use.